2024-05-11 15:27:10,427 - INFO - Test 'tests/test_FirstTaskPage.py::test_first_task_first' passed
2024-05-11 15:27:31,650 - ERROR - the link redirects to a third-party resource
2024-05-11 15:27:31,697 - ERROR - Test 'tests/test_FirstTaskPage.py::test_first_task_second' failed: browser = <selenium.webdriver.chrome.webdriver.WebDriver (session="cec6e96a330d01af2d4c1f498223197d")>
setup_logging = <Logger tests.conftest (DEBUG)>

    def test_first_task_second(browser, setup_logging):
        firsttaskpage = FirstTaskPage(browser)
        output = firsttaskpage.TextLinkOpen()
        # assert output[0] != None, setup_logging.error("link is invalid")
>       assert output[0] == output[1], setup_logging.error("the link redirects to a third-party resource")
E       AssertionError: None
E       assert 'https://tensor.ru/about' == 'https://tens.../devcenter/72'
E         
E         - https://tensor.ru/about/devcenter/72
E         ?                        -------------
E         + https://tensor.ru/about

tests\test_FirstTaskPage.py:13: AssertionError
